local_planner/src/nodes/planner_functions.cpp

```diff
@@ -289,13 +291,15 @@ double costFunction(int e, int z, const nav_msgs::GridCells& path_waypoints,
 
 void compressHistogramElevation(Histogram& new_hist,
                                 const Histogram& input_hist) {
-  int vertical_FOV_range_sensor = 20;
-  int lower_index = elevationAngletoIndex(
-      -(float)(vertical_FOV_range_sensor / 2.0), ALPHA_RES);
-  int upper_index = elevationAngletoIndex(
-      (float)(vertical_FOV_range_sensor / 2.0), ALPHA_RES);
-
-  for (int e = lower_index; e <= upper_index; e++) {
+  float vertical_FOV_range_sensor = 20.0;
+  PolarPoint p_pol_lower = {};
```

Comment:
Put the below initialization straight inside, with a dummy variable for the z value, so we don't have uninitialized members of our struct.